src/overlays/actors/ovl_Nameplate/ac_nameplate.c

```diff
@@ -2,33 +2,77 @@
 #include "m_actor_dlftbls.h"
 #include "m_object.h"
 #include "overlays/gamestates/ovl_play/m_play.h"
+#include "m_field_info.h"
+#include "m_player_lib.h"
+#include "m_npc.h"
+#include "m_msg_main.h"
+#include "69E2C0.h"
+#include "overlays/actors/player_actor/m_player.h"
 
-void func_80A963C0_jp(Actor* thisx, Game_Play* game_play);
-void func_80A963D0_jp(Actor* thisx, Game_Play* game_play);
+void aNP_actor_ct(Actor* thisx, Game_Play* game_play);
+void aNP_actor_dt(Actor* thisx, Game_Play* game_play);
+void aNP_set_talk_info(Actor* thisx);
+void aNP_actor_move(Actor* thisx, Game_Play* game_play);
 void aNP_actor_init(Actor* thisx, Game_Play* game_play);
 
-#if 0
 ActorProfile Nameplate_Profile = {
```

Comment:
Any reason we keep the comments in the profile lol, I've seen it also in toudai

Reply:
Afaik, it is meant as a formatting hack. Though from the few I've tried, the formatter actually doesn't change them without the dummy comments. Perhaps we can try removing them?